### PR TITLE
Support for relocation mapping files

### PR DESCRIPTION
**Pull request by glyn**
Fixes https://github.com/deislabs/duffle/issues/772

### CODE REVIEW

File: cmd/duffle/main.go [hunk 1]
```diff
@@ -111,7 +112,46 @@ func prepareDriver(driverName string) (driver.Driver, error) {
...
+
+type driverWithRelocationMapping struct {
+	driver     driver.Driver
+	relMapping string
+}
+
+func (d *driverWithRelocationMapping) Run(op *driver.Operation) error {
+	// if there is a relocation mapping, ensure it is mounted
+	if d.relMapping != "" {
+		fmt.Printf("mounting relocation mapping %q\n", d.relMapping)
```
Comment (technosophos) [inline cmd/duffle/main.go]: I believe this should be `fmt.Fprintf(os.Stdout...` or it should be removed.

File: cmd/duffle/relocate.go [hunk 2]
```diff
@@ -106,18 +107,14 @@ func (r *relocateCmd) run() error {
 		return err
 	}
 
-	return r.writeBundle(bun)
+	return nil
```
Comment (technosophos) [inline cmd/duffle/relocate.go]: This whole section (lines 106-110) should be abbreviated to `return r.relocate(bun)`.

File: cmd/duffle/main.go (hunk 1 above)
Comment (glyn) [reply cmd/duffle/main.go]: Oops, I forgot to delete that. Thanks for spotting this.

File: cmd/duffle/relocate.go (hunk 2 above)
Comment (glyn) [reply cmd/duffle/relocate.go]: Yup - another left-over I forgot to clean up. The linter didn't spot it.

**Pull request merged**